### ISSUE TITLE
Wrapper: Unable to get service from context

### ISSUE THREAD

**Issue by NeutronStein** [micro/go-plugins#98]
Unable to get service from context. Error is always returned.
```go
_, ok := micro.FromContext(ctx)
    if !ok {
        return errors.InternalServerError("service-name", "Could not retrieve service")
}
```
Wrappers have been set up correctly.
```go
svc.Init(
        micro.WrapClient(service.NewClientWrapper(svc)),
        micro.WrapHandler(service.NewHandlerWrapper(svc)),
)
```
Everything was working fine then broke without any code change.

**Comment by NeutronStein** [comment]
There was a change on go-micro side that made svc.Init() not processing options. I had to call it twice to make it work.

**Comment by asim** [comment]
Is it an issue? You closed this.

**Comment by NeutronStein** [comment]
[this](https://github.com/micro/go-micro/commit/b43985401a2d014dc1d1968c811f33de32087f96) commit i think broke service.Init() functionality. I had to call service.Init() twice (according to [this](https://github.com/micro/go-micro/commit/d7dc4e618bb890958da4bfcfe0475a8dc38f7c4f) commit) to make it work since on the first call of service.Init() the options are ignored. They are processed on second call. That is how I made it work again.
